scripts/st2bootstrap-deb.sh

```diff
@@ -96,6 +96,30 @@ setup_args() {
   fi
 }
 
+check_st2_host() {
+  # Check that the following TCP ports are available.
+  # Abort the installation early if the required ports are being used by an existing process.
+
+  # nginx (80, 443), mongodb (27017), rabbitmq (4369, 5672, 25672), and st2 (9100-9102).
```

Comment:
+1 for detailed note like that.

Maybe this information will be helpful in error message for users as well. So they'll know what ports to check in case of port conflict.

Just keep in mind that with PR like this we're trying to reduce support burden in #community and being a bit more specific may save our time in future.